bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -102,8 +104,14 @@
     )
     private final OpStatsLogger writeDataLatency;
 
+    protected Throttler replicationThrottle = null;
 
-    public LedgerFragmentReplicator(BookKeeper bkc, StatsLogger statsLogger) {
+    private int averageEntrySize;
+
+    private static final int initialAvgEntriesSize = 1024;
```

Comment:
uppercase?

Reply:
rethinking about this. can you please explain why we need this initial value ?

Reply:
Because, `replicationThrottle.acquire`  permits must be positive.
```
if (replicationThrottle != null) {
            replicationThrottle.acquire(averageEntrySize);
        }
```